src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,34 @@
+.calendar {
+  position: absolute;
+  top: 50%;
+  left: 50%;
```

Comment:
do not do this on a BEM block. you can create another class let's say for the `body`. and then make the calendar it's element